jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -22,7 +22,6 @@ def test_2d_parser(specviz2d_helper):
     dc_1 = specviz2d_helper.app.data_collection[1]
     assert dc_1.label == 'Spectrum 1D'
     assert 'header' not in dc_1.meta
-    assert dc_1.meta['NAXIS'] == 2

```

Comment:
as far as I can tell, we don't rely on this anywhere, and since this PR replaces the previous collapsing with extraction that does not set this meta field, I removed it from the test.  If the extraction should set NAXIS instead, we can do that and re-introduce this line in the test.